docs/synopsis.adoc

```diff
@@ -12,11 +12,11 @@
 * `get_completion_scheduler` - a tag used to retrieve a completion_scheduler from a sender's attributes
 
 ==== https://github.com/intel/cpp-baremetal-senders-and-receivers/blob/main/include/async/concepts.hpp[concepts.hpp]
+* `multishot_sender<S>` - a concept modelled by senders where `connect` may operate on lvalues
```

Comment:
question: sometimes i wonder if things being out of alphabetical order on a PR is a test to see how closely we are reviewing it.... XD